src/pages/admin/GlobalAuth.tsx

```diff
@@ -50,20 +64,43 @@ export const GlobalAuth = () => {
           </P1>
         </>
       )}
-      <Link to={authUrl as string} target="_blank">
-        <Button
-          text="Sign In"
-          data-testid="submit-button"
-          onClick={signIn}
-        ></Button>
-      </Link>
+      {authUrl && (
+        <Link to={authUrl} target="_blank">
+          <Button
+            text="Sign In"
+            data-testid="submit-button"
+            onClick={signIn}
+          ></Button>
+        </Link>
+      )}
       {!message ? (
         <div></div>
       ) : message.error ? (
         <H1 style={{ color: "red" }}>{message.error}</H1>
```

Comment:
please don't use hardcoded colors, use a theme from the context `theme.colors.red100`